### ISSUE TITLE
Mistake in Port_bindings documentation?

### ISSUE THREAD

**Issue by MichaelKueller** [docker/docker-py#1014]
In your port_bindings documentation it says: 

```
Similarly for several container ports bound to a single host port:

cli.create_host_config(port_bindings={
    1111: [1234, 4567]
})
```

Isn't it the other way around? Several host ports bound to a single container port?

**Comment by aanand** [comment]
Good catch - fixing it now.

**Comment by aanand** [comment]
Fixed in #1015
